repos/iiif-storage/config/service.config.php

```diff
@@ -403,11 +428,15 @@
             'top-contributors' => function (ContainerInterface $c) {
                 return $c->get(TopContributorsIngester::class);
             },
+            'html' => function (ContainerInterface $c) {
+                return $c->get(HtmlIngester::class);
+            },
         ],
     ],
     'media_renderers' => [
         'invokables' => [
             'iiif' => null,
+            'html' => null,
```

Comment:
Why is this needed? I see we're replacing it as a factory, does an invokable take precedence?

Reply:
Possibly, I remember running into issues with the IIIF one, also to remove any ambiguity for anyone debugging. (Quick scan of Zend docs didn't produce an answer either)